clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -254,7 +254,7 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
       }
       // Specifically check if the math library function corresponding to this
       // builtin is supported for SYCL
-      unsigned BuiltinID = (Callee ? Callee->getBuiltinID() : 0);
```

Comment:
Just curious, what was wrong here?

Reply:
Static code analyzer somehow worried that Callee is checked on nullptr here however is dereferenced earlier at line 235 without  this check.
But actually it is already tested at line 216 and no additional check is required.

Reply:
Oh, I missed resetting Callee at line 217.